.releaserc

```diff
@@ -1,17 +1,19 @@
 {
   "branch": "master",
   "tagFormat": "java-sdk-${version}",
+  "debug": true,
   "plugins": [
+    "@semantic-release/changelog",
```

Comment:
In the go and python cores, I didn't specify a "plugins" field.  Instead, the list of plugins are listed in the "prepare" field.
I don't know what the difference is, but I do know that those two repos seems to be working correctly now :)
But the key is to ensure that the "git" plugin is executed last I think after the steps that create commits are finished.
Here's a link to the go core .releaserc file in case you want to take a look at that for comparison purposes:
https://github.com/IBM/go-sdk-core/blob/master/.releaserc

Reply:
Yeah I saw that you were doing it a bit differently. The only reason I went with the "plugins" approach is that it seems to be the new standard way to define it, at least according to semantic-release documentation: https://github.com/semantic-release/semantic-release/blob/master/docs/usage/configuration.md#options

Since we're running `npx semantic-release`, we should be grabbing the latest version. I'm just a bit worried that some build in the future might grab a major release and remove support for what seems to be the old way of defining things. I tried to still re-arrange the order to match what you've done since that was presumably the issue.